README.md

```diff
@@ -149,6 +149,7 @@ Alternatively, you can use an existing Zowe CLI profile instead:
 
 Currently, the Zowe Python SDK supports the following interfaces:
 
+- z/OSMF Workflows
```

Comment:
(nitpick) Could we move this to the bottom? 

My intention is to try to keep things in the order in which they were added. However, I don't thikn Console commands were the first thing to be added...
Perhaps we just sort them alphabetically 🙏